x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/hooks/use_latest_vulnerabilities.tsx

```diff
@@ -91,7 +92,7 @@ export const useLatestVulnerabilities = (options: VulnerabilitiesQuery) => {
    * the last loaded record to be used as a from parameter to fetch the next chunk of data.
    */
   return useInfiniteQuery(
-    [LATEST_VULNERABILITIES_INDEX_PATTERN, options],
+    [CDR_LATEST_NATIVE_VULNERABILITIES_INDEX_PATTERN, options],
```

Comment:
should it also be ` CDR_VULNERABILITIES_INDEX_PATTERN` here?

Reply:
no, you mentioned in the previous comment that we will leave the detection rule creation for another ticket. 
`CDR_LATEST_NATIVE_VULNERABILITIES_INDEX_PATTERN` points to the same value as we had before.